packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -134,13 +134,59 @@ export type ScopedSchemaName<
  *
+ * @remarks
+ * All schema produced by this factory get a {@link TreeNodeSchemaCore.identifier|unique identifier} by {@link ScopedSchemaName|combining} the {@link SchemaFactory.scope} with the schema's `Name`.
+ * The `Name` part may be explicitly provided as a parameter, or inferred as a structural combination of the provided types.
+ * The APIs which use this second approach, structural naming, also deduplicate all equivalent calls.
+ * Therefor two calls to `array(allowedTypes)` with the same allowedTypes will return the same {@link TreeNodeSchema} instance.
+ * On the other hand, two calls to `array(name, allowedTypes)` will always return different {@link TreeNodeSchema} instances
+ * and it is an error to use both in the same tree (since their identifiers are not unique).
+ *
+ * Note:
+ * POJO stands for Plain Old JavaScript Object.
+ * This means an object that works like a `{}` style object literal.
+ * In this case it means the prototype is `Object.prototype` and acts like a set of key value pairs (data, not methods).
+ * The usage below generalizes this to include array and map like objects as well.
+ *
+ * There are two ways to use these APIs:
+ * |                     | Customizable | POJO Emulation |
+ * | ------------------- | ------------ |--------------- |
+ * | Declaration         | `class X extends schemaFactory.object("x", {}) {}` | `const X = schemaFactory.object("x", {}); type X = NodeFromSchema<typeof X>; `
+ * | Allows adding "local" (non-persisted) members | Yes. Members (including methods) can be added to class.        | No. Attempting to set non-field members will error. |
+ * | Prototype | The user defined class | `Object.prototype`, `Map.prototype` or `Array.prototype` depending on node kind |
+ * | Structurally named Schema | Not Supported | Supported |
+ * | Explicitly named Objects | Supported | Supported |
+ * | Explicitly named Maps and Arrays | Supported: Both declaration approaches can be used | Not Supported |
+ * | node.js assert.deepEqual | Compares like class instances: equal to other nodes of the same type with the same content, including custom local fields. | Compares like plain objects: equal to plain JavaScript objects with the same fields, and other nodes with the same fields, even if the types are different. |
+ * | Intellisense | Shows and links to user defined class by name: `X` | Shows internal type generation logic: `object & TreeNode & ObjectFromSchemaRecord<{}> & WithType<"test.x">` |
+ * | Recursion | Supported with special declaration patterns. | Unsupported: Generated d.ts files replace recursive references with `any`, breaking use of recursive schema across compilation boundaries |
+ *
+ * Note that while "POJO Emulation" nodes act a lot like POJO objects, they are not true POJO objects:
+ *
+ * - Adding new arbitrary fields will error, as well some cases of invalid edits.
```

Comment:
Note: if you want this to render as a list in intellisense / generated docs, you'll need a line break between each bullet.
https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#line-breaks

Reply:
thats a generated docs specific thing. Works fine in IntelliSense is as. Fill fix though.

Reply:
Really? Nice! Last time I tried this it put everything on the same line.

Reply:
I believe you need a line break before the list (which I had)